### DISCUSSION TITLE
Serenity.EntityGrid can not get detail grid data

### DISCUSSION THREAD

**Discussion by wow-prime** [serenity-is/Serenity#6150] [Help]
If i use "Serenity.Extensions.GridEditorBase" in my editor, my left join column do not show in grid.

![1639647189819](https://user-images.githubusercontent.com/19904096/146346056-7409ba99-3a86-44b1-8f01-745afb939e27.jpg)

In this case, i can get detail grid data when i click update button.
![1639647493148](https://user-images.githubusercontent.com/19904096/146346893-1da29c1f-12b9-45b8-8f20-f24418c1356a.jpg)

If i use "Serenity.EntityGrid" in my editor, my left join column show in grid
![1639647754497](https://user-images.githubusercontent.com/19904096/146347288-8a71e463-b8df-433e-8c4d-22781739cfeb.jpg)

In this case, i can not get detail grid data when i click update button.
![1639647842904](https://user-images.githubusercontent.com/19904096/146347540-f5b62e98-b658-49e5-a201-bd1a5282de65.jpg)

My ForeignKey setting:
![1639647944759](https://user-images.githubusercontent.com/19904096/146347824-d1ab2d08-3dcd-4ca3-bc45-4b79295e79d0.jpg)

How can I display in grid and get the value in server side at the same time?

**Comment by wezmag** [comment]
You should have a field with `MasterDetailRelation` attribute, try add `IncludeColumns`  with "OrgName".
or `ColumnSelection = Serenity.Services.ColumnSelection.Details`
For example:
``` csharp
[MasterDetailRelation(nameof(GroupMemberRow.RowFields.Id), IncludeColumns = "OrgName")]
public List<GroupMemberRow> MemberList
{
   ...
}
```

``` csharp
[MasterDetailRelation(nameof(GroupMemberRow.RowFields.Id), ColumnSelection = Serenity.Services.ColumnSelection.Details)]
public List<GroupMemberRow> MemberList
{
   ...
}
```

**Reply by wezmag** [reply]
BTW, You need to use `Serenity.Extensions.GridEditorBase`, otherwise you will not get the value in server side when submit data to backend.
